Tools/clinic/clinic.py

```diff
@@ -5212,10 +5321,33 @@ def parse_converter(
                     "Annotations must be either a name, a function call, or a string."
                 )
 
+    def parse_deprecated_positional(self, thenceforth: str) -> None:
+        assert isinstance(self.function, Function)
+        fname = self.function.full_name
+
+        if "." not in thenceforth:
+            fail(
+                f"Function {fname!r}: '* [from ...]' format expected to be "
+                f"'<major.minor>', where 'major' and 'minor' are digits."
+            )
+        if self.keyword_only:
+            fail(f"Function {fname!r}: '* [from ...]' must come before '*'")
+        if self.deprecated_positional:
+            fail(f"Function {fname!r} uses '[from ...]' more than once.")
+        try:
+            major, minor = thenceforth.split(".")
+            self.deprecated_positional = int(major), int(minor)
+        except ValueError:
+            fail(
+                f"Function {fname!r}, '* [from <major.minor>]': "
+                f"'major' and 'minor' must be digits, not {thenceforth!r}"
```

Comment:
"digits"? Maybe "integers" or "integer numbers" or "decimal integers"?

Reply:
Perhaps integers is better.